src/Serilog.Sinks.MongoDB/Sinks/MongoDB/MongoDBSink.cs

```diff
@@ -35,6 +36,11 @@ public class MongoDBSink : PeriodicBatchingSink
         readonly IMongoDatabase _mongoDatabase;
 
         /// <summary>
+        /// The default name for the log collection.
```

Comment:
Any reason to move this? (Just helps to keep the diff focused if cosmetic changes are limited).